pkg/cedar/swagger.json

```diff
@@ -0,0 +1,241 @@
+        "software_product_name" : {
+          "type" : "string"
+        },
+        "software_manufacture" : {
+          "type" : "string"
+        },
+        "software_product_type" : {
+          "type" : "string"
+        },
+        "software_product_version" : {
+          "type" : "string"
+        }
+      }
+    },
+    "mission_essential_function" : {
+      "type" : "object",
+      "required" : [ "mission_essential_function_level", "mission_essential_function_name" ],
+      "properties" : {
+        "mission_essential_function_level" : {
+          "type" : "string"
+        },
+        "mission_essential_function_name" : {
+          "type" : "string"
+        }
+      }
+    },
+    "systemid_GET_response" : {
+      "type" : "object",
+      "required" : [ "SystemDetail" ],
+      "properties" : {
+        "SystemDetail" : {
+          "$ref" : "#/definitions/SystemDetail"
+        }
+      }
+    },
+    "SystemDetail" : {
+      "type" : "object",
+      "required" : [ "id", "mission_essential_function", "software", "system_acronym", "system_name" ],
+      "properties" : {
+        "business_function_alignment" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        },
+        "supported_user_count" : {
+          "type" : "integer",
+          "format" : "int32"
+        },
+        "business_owner" : {
+          "type" : "string"
+        },
+        "business_owner_org" : {
+          "type" : "string"
+        },
+        "data_categories" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        },
+        "mission_essential_functions" : {
+          "type" : "string"
+        },
+        "csp_hosted" : {
+          "type" : "boolean"
+        },
+        "software" : {
+          "type" : "array",
+          "items" : {
+            "$ref" : "#/definitions/software"
+          }
+        },
+        "product_owner" : {
+          "type" : "string"
+        },
+        "contractor_ftes" : {
+          "type" : "integer",
+          "format" : "int32"
+        },
+        "gpr_changes_next_release" : {
+          "type" : "boolean"
+        },
+        "system_maintainer_org" : {
+          "type" : "string"
+        },
+        "csp_service_model_type" : {
+          "type" : "string"
+        },
+        "next_major_project_date" : {
+          "type" : "string"
+        },
+        "system_owner" : {
+          "type" : "string"
+        },
+        "tlc_profile_reviewer" : {
+          "type" : "string"
+        },
+        "id" : {
+          "type" : "string"
+        },
+        "system_acronym" : {
+          "type" : "string"
+        },
+        "mission_essential_function" : {
+          "type" : "array",
+          "items" : {
+            "$ref" : "#/definitions/mission_essential_function"
+          }
+        },
+        "next_planned_prod_release_date" : {
+          "type" : "string"
```

Comment:
I can ask if there's a way to update this. It might be the case the generation isn't that detailed. We might just need to find out from the CEDAR team.